doc/getting-started/building-from-source.rst

```diff
@@ -149,7 +151,11 @@ HYBRID_RT:
    pre-launched RTVM, one pre-launched Service VM, and one post-launched
    Standard VM.
 
-Assuming that you are at the top level of the acrn-hypervisor directory, perform the following:
+XML configuration files for those scenarios on supported boards are available
+under the ``misc/config_tool/data`` directory.
+
+Assuming that you are at the top level of the acrn-hypervisor directory, perform
+the following to build the hypervisor, device model and tools:
 
 .. note::
    The release version is built by default, ``RELEASE=0`` builds the debug version.
```

Comment:
The ``whl-ipc-i7/hybrid_rt.xml`` explicitly sets ``RELEASE=n``  and the default value for ``RELEASE`` in the config.xsd file is also ``n``, so it would appear the default is now to build a debug image.    Also, does the make command need to say ``RELEASE=n`` now instead of ``RELEASE=0``?

Reply:
All the predefined scenario XMLs specify `RELEASE=n`. There is another default setting in the Makefile (which is `y`), but considering the fact that we require a scenario XML to kick off the build and `RELEASE` is mandatory in that XML file, I don't think the one in Makefile ever takes effect. I'll update the doc accordingly.

I think `RELEASE=y|n` is our preference now while we have kept `RELEASE=0|1` for backward compatibility. I'll update the doc to use our latest preference as well.

Reply:
In this PR I'll only update the occurences of the items above in `building-from-source.rst`. There are also other places using `RELEASE=0|1` but they are all tutorials using a specific version of ACRN.

Reply:
> I think `RELEASE=y|n` is our preference now while we have kept `RELEASE=0|1` for backward compatibility. I'll update the doc to use our latest preference as well. The logic seems to work as expected here. I think it's a good idea to indicate [0|1] is our preference nowadays.

There is however a problem with what gets printed out at the end of the build (and recorded in the `build/hypervisor/cfg.log` file). It **nearly always** reports a `BUILD type = RELEASE` **except** when using `make RELEASE=0`. The problem is in [this line](https://github.com/projectacrn/acrn-hypervisor/blob/master/Makefile#L102) and the value of `RELEASE` is not propagated correctly.

Here is a little experiment I ran today
* Using default values:
```
make
...
ACRN Configuration Summary:
BOARD = nuc7i7dnb        SCENARIO = industry
BUILD type = RELEASE
VM configuration is based on:
        Source code at:                 /home/gvancuts/work/acrn-hypervisor/build/hypervisor/configs

$ grep RELEASE build/ -r
build/hypervisor/cfg.log:BUILD type = RELEASE
build/hypervisor/configs/config.mk:CONFIG_RELEASE=n
build/hypervisor/.scenario.xml:        <RELEASE>n</RELEASE>
```
* Setting `RELEASE` to `y` and `1`:
```
--------------
make RELEASE=y
...
ACRN Configuration Summary:
BOARD = nuc7i7dnb        SCENARIO = industry
BUILD type = RELEASE
VM configuration is based on:
        Source code at:                 /home/gvancuts/work/acrn-hypervisor/build/hypervisor/configs

$ grep RELEASE build/ -r
build/hypervisor/cfg.log:BUILD type = RELEASE
build/hypervisor/configs/config.mk:CONFIG_RELEASE=y
build/hypervisor/configs/scenarios/industry/nuc7i7dnb.config:CONFIG_RELEASE=y
build/hypervisor/include/config.h:#define CONFIG_RELEASE 1
build/hypervisor/.scenario.xml:        <RELEASE>y</RELEASE>

--------------
make RELEASE=1
...
ACRN Configuration Summary:
BOARD = nuc7i7dnb        SCENARIO = industry
BUILD type = RELEASE
VM configuration is based on:
        Source code at:                 /home/gvancuts/work/acrn-hypervisor/build/hypervisor/configs

$ grep RELEASE build/ -r
build/hypervisor/cfg.log:BUILD type = RELEASE
build/hypervisor/configs/config.mk:CONFIG_RELEASE=y
build/hypervisor/configs/scenarios/industry/nuc7i7dnb.config:CONFIG_RELEASE=y
build/hypervisor/include/config.h:#define CONFIG_RELEASE 1
build/hypervisor/.scenario.xml:        <RELEASE>y</RELEASE>
```
* Setting `RELEASE` to `n` and `0`:
```
--------------
make RELEASE=n
...
ACRN Configuration Summary:
BOARD = nuc7i7dnb        SCENARIO = industry
BUILD type = RELEASE
VM configuration is based on:
        Source code at:                 /home/gvancuts/work/acrn-hypervisor/build/hypervisor/configs

$ grep RELEASE build/ -r
build/hypervisor/cfg.log:BUILD type = RELEASE
build/hypervisor/configs/config.mk:CONFIG_RELEASE=n
build/hypervisor/.scenario.xml:        <RELEASE>n</RELEASE>

--------------
make RELEASE=0
...
ACRN Configuration Summary:
BOARD = nuc7i7dnb        SCENARIO = industry
BUILD type = DEBUG
VM configuration is based on:
        Source code at:                 /home/gvancuts/work/acrn-hypervisor/build/hypervisor/configs

$ grep RELEASE build/ -r
build/hypervisor/configs/config.mk:CONFIG_RELEASE=n
build/hypervisor/.scenario.xml:        <RELEASE>n</RELEASE>

```

Reply:
> > I think `RELEASE=y|n` is our preference now while we have kept `RELEASE=0|1` for backward compatibility. I'll update the doc to use our latest preference as well. The logic seems to work as expected here.
> 
> I think it's a good idea to indicate [n|y] is our preference nowadays.

On second thought, I don't think this is quite ready yet - see #5772 . Unless you think we can fix this on time for v2.4, my recommendation is that we stick to using `[0|1]`. And we might even want to include a note to explain that the `RELEASE` value from the scenario XML file is not properly taken into account for all components...

Reply:
With #5791 merged, I'll update the example build commands here to use ``RELEASE=y``.